docs/use-the-network/light-hotspots/light-hotspots.mdx

```diff
@@ -172,6 +172,8 @@ All Hotspots will not need to follow the Blockchain.
 - [Kerlink gateways](/mine-hnt/light-hotspots/guides/kerlink)
   - Kerlink gateways will only mine HNT as Data Only Hotspot unless it is
     specified as a Helium-Compatible Kerlink Miner.
+- [Cotx gateways](/docs/use-the-network/light-hotspots/guides/cotx.mdx)
```

Comment:
```suggestion
- [Cotx gateways](/use-the-network/light-hotspots/guides/cotx)
```

I think path and extension need to be removed. You can also move it to the same folder as the Dragino guide btw

Reply:
> I think path and extension need to be removed. You can also move it to the same folder as the Dragino guide btw

OK, the question has been modified.